lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -403,7 +403,7 @@ def add_level_three(post, options)
         post[:level3] = level_three unless level_three.empty?
       end
 
-      def add_expand_parameters(post, options)
+      def add_expand_parameters(post, options, method, url)
```

Comment:
`method` and `url` were added but never used

Reply:
I does get used [here](https://github.com/activemerchant/active_merchant/pull/5360/files#diff-19492101b546175aa3ca6a16ec7d0a610d0adae03cbaf130eff54092dec8e0fbR327) which is why we need to pass them